dvb/cx23885/files/udev-dvb.rules

```diff
@@ -0,0 +1 @@
+SUBSYSTEM=="dvb", GROUP="44", MODE="0660"
```

Comment:
there should be no rules like that, Talos doesn't have any users/groups at all.

any access to the devices should be provided via Kubernetes device plugins

Reply:
There already is in the existing udev rules as this one is basically the same as one of the defaults except for the video part being in numeric form.

I wouldn't even know how a device plugin could fix the permissions of the device nodes.